### PR TITLE
Update OBS

### PR DESCRIPTION
**Pull request by aza547**
Remaining work:
- Write UTs? (lol)

### CODE REVIEW

File: src/main/recorder.ts [hunk 1]
```diff
@@ -482,6 +499,42 @@ class Recorder {
 
     console.debug('[OBS] Asserted OBS signal:', value);
   };
+
+  static getInputAudioDevices() {
+    // @@@ todo check init or fail
+    const inputDevices: IDevice[] = [];
+
+    (osn.NodeObs.OBS_settings_getInputAudioDevices() as IOBSDevice[]).forEach(
```
Comment (tdaugaard) [inline src/main/recorder.ts:500]: Would look slightly cleaner as an assignment of `const inputDevices =  (osn.NodeObs.OBS_settings_getInputAudioDevices() as IOBSDevice[]).map(...)` I think as that'd make it immediately obvious what's going on.

Comment (aza547) [reply src/main/recorder.ts]: Agree this was super ugly - I think I've gone one step further and removed all the map stuff entirely as it was to add a parameter that was unused! This code was copy pasted straight out of the streamlabs desktop repository... much happier with it now as:

```
  getInputAudioDevices() {
    console.info('[Recorder] Getting available input devices');

    if (!this.obsInitialized) {
      throw new Error('[Recorder] OBS not initialized');
    }

    const inputDevices =
      osn.NodeObs.OBS_settings_getInputAudioDevices() as IOBSDevice[];

    return inputDevices.filter((v) => v.id !== 'default');
  }
```

**Comment by aza547** [comment]
Some old config for future reference:

```
{
	"maxStorage": 50,
	"minEncounterDuration": 15,
	"audioInputDevice": "all",
	"audioOutputDevice": "all",
	"startUp": true,
	"startMinimized": false,
	"recordRetail": true,
	"recordClassic": true,
	"recordRaids": true,
	"recordDungeons": true,
	"recordTwoVTwo": true,
	"recordThreeVThree": true,
	"recordSkirmish": true,
	"recordSoloShuffle": true,
	"recordBattlegrounds": true,
	"obsOutputResolution": "2000x1000",
	"obsFPS": "60",
	"obsKBitRate": 15,
	"selectedCategory": 1,
	"obsCaptureMode": "game_capture",
	"obsRecEncoder": "auto",
	"recordFiveVFive": true,
	"retailLogPath": "D:\\World of Warcraft\\_retail_\\Logs",
	"classicLogPath": "D:\\World of Warcraft\\_classic_\\Logs",
	"storagePath": "D:\\_wow-recorder",
	"bufferStoragePath": "D:\\wow-recorder\\.temp",
	"monitorIndex": 1
}
```

**Comment by aza547** [comment]
I think this is all ready to go (except UTs...)

**Pull request merged**